### PR TITLE
Add raw task for installing Python 2

### PR DESCRIPTION
**Pull request by itsjeyd**
... and enable it for `mongodb` hosts.

Also, clean up this repo's README a bit.

Part of the scope of [OC-2773](https://tasks.opencraft.com/browse/OC-2773).

**Related PRs**

- https://github.com/open-craft/deployment-secrets-databases/pull/19

**Test instructions**

See parent PR (https://github.com/open-craft/deployment-secrets-databases/pull/19).

**Reviewers**

- [ ] @UmanShahzad

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,49 +1,47 @@
...
-3. Create instance using Ubuntu 14.04 image. We suggest you use vanilla image from 
-   [ubuntu page](https://cloud-images.ubuntu.com/). This image might be small, mysql 
+3. Create an instance using an appropriate image, such as Ubuntu 16.04. We suggest you use vanilla image from
+   [ubuntu page](https://cloud-images.ubuntu.com/). This image might be small; MySQL
    data will be stored elsewhere.
-4. Attach the data volume to the vm (go to `Volumes` -> your data volume -> this down `v` arrow near Edit Volume -> 
-   `Edit Attachments`).    
-5. Manual step: login to administrative user of the database machine and format /dev/vdb using ext4. 
-   To do this login to the instance and issue `fdisk /dev/vdb` and then issue:    
+4. Attach the data volume that you created earlier to the VM (go to `Volumes` -> <data volume> -> expand dropdown next to "Edit Volume" ->
```
Comment (UmanShahzad) [inline README.md]: In the [rich diff view](https://github.com/open-craft/deployment-deploy-databases/pull/9/files?short_path=04c6e90#diff-04c6e90faac2675aa89e2176d2eec7d8), this "<data volume>" is being hidden. It also is being hidden in the [actual rendered page](https://github.com/open-craft/deployment-deploy-databases/blob/e38c3238d794205a7c2538cb91b0a32ea1dbbb55/README.md#create-an-openstack-environment).

**Comment by itsjeyd** [comment]
Rebased from e38c3238d794205a7c2538cb91b0a32ea1dbbb55.

File: README.md (hunk 1 above)
Comment (itsjeyd) [reply README.md]: @UmanShahzad Thanks for catching that! Fixed.

**Comment by itsjeyd** [comment]
Thanks @UmanShahzad!

**Pull request merged**